backend/tracim_backend/lib/utils/authentication.py

```diff
@@ -177,15 +199,29 @@ def _acs(self, request: TracimRequest) -> Response:
         except ResponseLifetimeExceed as e:
             return Response(e.__str__())
         identity = authn_response.get_identity()
-        if identity is None or "UserID" not in identity:
+        if identity is None:
             return HTTPBadRequest()
+        idp_name = request.POST.get("RelayState")
```

Comment:
what is `RelayState`?

Reply:
RelayState is information conveyed through the auth process, this is here to keep info about at which idp the user authenticated. To then get the proper attribute bindings. This usage is standard to SAML

Reply:
added details in comments